services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/models/schemas/meta.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, ClassVar
+
+from models_library.basic_types import VersionStr
+from pydantic import BaseModel, HttpUrl
+
+
+class Meta(BaseModel):
```

Comment:
THOUGHT. This is a model that at some point we might want to put in common so we do not copy it again and again.